model/evaluation/metrics.py

```diff
@@ -0,0 +1,52 @@
+from tensorflow.keras import backend as K
```

Comment:
This file is not being used for now. The only metric that did work was accuracy. The others (even those implemented by keras) generated bugs.